[terra-table] Rows and Header disabled checkbox accessibility fix (#3024)
cerner/terra-core: packages/terra-table/src/terra-dev-site/doc/table/guides.5/SingleSelectTable.1.doc.mdx

+ const createCellsForRow = cells => cells.map(cell => createCell(cell));
```
Finally we can implement a method to loop through our data and create the table with our methods and call it from our render method.

> Additionally whenever the table is rendered without a `check mark`, for instance when rowStyle="toggle" it is recommended to set isDisabled=true

Reviewer: ```suggestion
Additionally, set `isDisabled=true` within the `selectAllColumn` property whenever the table is rendered without a checkmark. An example is when `rowStyle="toggle"`.
```
